Makefile

```diff
@@ -571,9 +571,13 @@ verify-ui-bindata: ## Run verification for UI bindata
 
 .PHONY: cobra-docs
 cobra-docs:
-	TANZU_CLI_NO_INIT=true TANZU_CLI_NO_COLOR=true $(GO) run ./cmd/cli/tanzu generate-all-docs
+	#TODO: Currenyly generating docs into both cli/core/docs/cli/commands and docs/cli/commands directories. Need to change it when core CLI moved to different repo
+	cd cli/core && TANZU_CLI_NO_INIT=true TANZU_CLI_NO_COLOR=true $(GO) run ./cmd/tanzu generate-all-docs
+	sed -i.bak -E 's/\/[A-Za-z]*\/([a-z]*)\/.config\/tanzu\/pinniped\/sessions.yaml/~\/.config\/tanzu\/pinniped\/sessions.yaml/g' cli/core/docs/cli/commands/tanzu_pinniped-auth_login.md
+	cd cli/core && TANZU_CLI_NO_INIT=true TANZU_CLI_NO_COLOR=true $(GO) run ./cmd/tanzu generate-all-docs --docs-dir "../../docs/cli/commands"
```

Comment:
We should not add plugin specific command details in `cli/core/docs/cli/commands`.

My preference would be not to include anything in the `cli/core` for now and let's only update in `docs/cli/commands` to fix the issue.

Reply:
I have the same opinion. Updated.